src/manager.cpp

```diff
@@ -347,8 +347,85 @@ types::DbusVariantType
 void Manager::collectSingleFruVpd(
```

Comment:
The code looks ok to me. Just that the doxygen needs to be modified as per the implementation where the flag is not being used.

Reply:
i can add a todo in code to add CM flag check. so that doxygen is not required to modify. is that fine?

Reply:
done